include/utility/json_utility.hpp

```diff
@@ -711,7 +711,8 @@ inline bool isActionRequired(const nlohmann::json& i_sysCfgJsonObj,
         return false;
     }
 
-    if (i_sysCfgJsonObj.empty() || !(i_sysCfgJsonObj.contains("frus")))
+    if (i_sysCfgJsonObj.empty() || !(i_sysCfgJsonObj.contains("frus")) ||
```

Comment:
Yes, I think we can optimize this part. Also the log is not valid in case fru path is not found.
Please handle that as a different case and return appropriate message.